docs/asciidoc/http-client.adoc

```diff
@@ -421,7 +426,7 @@ Without a conversion to a template-like form, each distinct URI leads to the cre
 NOTE: Always apply an upper limit for the meters with URI tags. Configuring an upper limit on the number of meters can help in cases when the real URIs cannot be templated.
 You can find more information at https://micrometer.io/docs/concepts#_denyaccept_meters[`maximumAllowableTags`].
 
-When HTTP client metrics are needed for an integration with a system other than `Micrometer` or you want
+When HTTP client metrics are needed for an integration with a system other than `Micrometer`, or you want
```

Comment:
Remove the comma in this line. (The list has only two items, so you don't need a comma in it.)